### ISSUE TITLE
C intensity data: use scientific rather than political source

### ISSUE THREAD

**Issue by ElkeBrandes** [corradio/electricitymap#288]
This app is fantastic, but unfortunately, the data that are shown are biased and don't reflect the real energy intensity of nuclear power. Here is [an article](http://www.theecologist.org/News/news_analysis/2736691/false_solution_nuclear_power_is_not_low_carbon.html) on life cycle assessment (LCA) studies of different electricity sources. It contains many references of (more or less) rigorous LCA meta-analyses for renewable and nuclear energy. I would recommend taking the data from some of those scientific peer-reviewed sources rather than from a political panel such as the IPCC. 
Bottom line: The extremely high uncertainties of the carbon intensity of nuclear power make it difficult to put an number to it - for the most energy intensive processes, namely plant decommission and waste management, lots of assumptions produce high variability in predictions. The IPCC analysis, however, simply ignores fuel production, plant decommission and waste management, and therefore is favorably biased toward nuclear in comparison to many renewable forms of electricity.

**Comment by martindaniel4** [comment]
Thanks for your feedbacks @ElkeBrandes. I think it's great to have this discussion as this is a recurrent topic. 

@brunolajoie @corradio @ThierryOllivero - what are your thoughts on this ?

**Comment by ThierryOllivero** [comment]
Dear Elke,

Thanks for your feedback on the map. 

We took IPCC because we strongly believe that it is as of today the best independent source for emission factors. I do not believe that "The Ecologist" is an independent and unbiased source that could efficiently replace IPCC.  

This being said, the most pessimistic of the sources in your article gives values of 34 for wind, 49.9 for solar and 66 for nuclear (all gCO2/kWh). This clearly shows that Nuclear is a low-carbon technology.

We, at Tomorrow, are not politically directed towards a technology or another. We are not putting nuclear against renewable, and we would like to have everyone fighting to promote low-carbon technologies instead of coal, gas or oil. 

The IPCC, as written in their report here: https://www.ipcc.ch/pdf/assessment-report/ar5/wg3/ipcc_wg3_ar5_annex-iii.pdf , is taking into account supply-chain and infrastructure costs in their LCA analysis. 

I am neither for or against nuclear energy and I confirm that we do not show on this map the potential impacts of nuclear waste on the environnement. This map only shows, as stated in the header, the "Live CO2 emissions of the European electricity consumption". Nothing more. Nothing less. 

This being said, if you have any data to support the fact that plant decommission and waste management are particularly energy intensive, sufficiently to have an impact on the emission factors, I would be quite interested. 

Finally, I put below a reminder of what IPCC is: a group of scientists who provides science-based facts to help government design the best policies to protect our environment:

> The Intergovernmental Panel on Climate Change (IPCC) is the international body for assessing
the science related to climate change. The IPCC was set up in 1988 by the World Meteorological
Organization (WMO) and United Nations Environment Programme (UNEP) to provide policymakers
with regular assessments of the scientific basis of climate change, its impacts and future risks, and
options for adaptation and mitigation.
IPCC assessments provide a scientific basis for governments at all levels to develop climaterelated
policies, and they underlie negotiations at the UN Climate Conference – the United Nations
Framework Convention on Climate Change (UNFCCC). The assessments are policy-relevant but
not policy-prescriptive: they may present projections of future climate change based on different
scenarios and the risks that climate change poses and discuss the implications of response options,
but they do not tell policymakers what actions to take.
The IPCC embodies a unique opportunity to provide rigorous and balanced scientific information to
decision-makers because of its scientific and intergovernmental nature. Participation in the IPCC is
open to all member countries of the WMO and United Nations. It currently has 195 members. The
Panel, made up of representatives of the member states, meets in Plenary Sessions to take major
decisions. The IPCC Bureau, elected by member governments, provides guidance to the Panel on the
scientific and technical aspects of the Panel’s work and advises the Panel on related management
and strategic issues1
.
IPCC assessments are written by hundreds of leading scientists who volunteer their time and
expertise as Coordinating Lead Authors and Lead Authors of the reports. They enlist hundreds of
other experts as Contributing Authors to provide complementary expertise in specific areas.
IPCC reports undergo multiple rounds of drafting and review to ensure they are comprehensive and
objective and produced in an open and transparent way. Thousands of other experts contribute to
the reports by acting as reviewers, ensuring the reports reflect the full range of views in the scientific
community. Teams of Review Editors provide a thorough monitoring mechanism for making sure
that review comments are addressed. (For more details see IPCC Factsheet – How does the IPCC
select its authors? and IPCC Factsheet – How does the IPCC review process work?).

Best regards,

Thierry Ollivero

**Comment by ThierryOllivero** [comment]
I close this issue since we do not plan to change the emission factor source at the moment. Do not hesitate to reopen it if you have a better scientific source than IPCC.

The discussion can continue even if the issue is closed.

Thierry

**Comment by ElkeBrandes** [comment]
Dear Thierry, Daniel, and Tomorrow-Team,

First and foremost, I'd like to emphasize the strengths of this app, I think it is a great tool to show immediate effects of renewable energy, especially wind and solar, and helps raising awareness of the different climate policies of European countries. I also appreciate your openness for discussion. 
I’ll respond to Thierry’s comments one by one, in order to clarify my points and intention.

> We took IPCC because we strongly believe that it is as of today the best independent source for emission factors. I do not believe that "The Ecologist" is an independent and unbiased source that could efficiently replace IPCC.

Frankly, to believe that the IPCC is an independent scientific source is a misconception. IPCC’s working groups are nominated by its member governments. As you cited above, the IPCC _assesses_ the science. The reports are drafted by scientists and then reviewed and accepted by government delegates. One can argue that even peer-reviewed scientific publications are not totally independent, but at least they are not subject to governmental review. 

I am not aware of any red flags in the article in The Ecologist that I referred to, but I’d like to learn from you if you know of any political or private interests that are driving the author’s methods or conclusions.

I have looked into the IPCC reports to find out where the number of 12 g CO2eq/kWh comes from. They are citing two sources: Lenzen 2008, who showed a range of 10-130 g CO2eq/kWh from the meta-analysis and Warner and Heath 2012 who calculated a median of 12 g CO2eq/kWh in a harmonized meta-analysis. This latter study was criticized by Keith Barnham in the Ecologist article for a methodological flaw: the median stems from 99 scenarios of 27 studies, so many scenarios were based on the same assumptions and are therefore not independent from each other, creating a biased median. Apart from this flaw, and looking into other meta-analyses, it becomes obvious that this number is on the low end of a wide range of possible results. 

> This being said, the most pessimistic of the sources in your article gives values of 34 for wind, 49.9 for solar and 66 for nuclear (all gCO2/kWh). This clearly shows that Nuclear is a low-carbon technology.

After I have skimmed a few of the meta-analyses, the only conclusion I can draw is that there is scientific evidence for tremendous uncertainty about the life cycle carbon intensity of nuclear power (e.g., Lenzen 2008, Sovacool 2008, Warner and Heath 2012).

> The IPCC, as written in their report here: https://www.ipcc.ch/pdf/assessment-report/ar5/wg3/ipcc_wg3_ar5_annex-iii.pdf , is taking into account supply-chain and infrastructure costs in their LCA analysis.

Just to clarify: The IPCC is not doing any LCA analyses on nuclear power. They referred to _selected_ peer reviewed scientific literature, but they did not do any systematic reviews of existing studies. Of course, these same flaws might underlie the other electricity sources as well.

> I am neither for or against nuclear energy and I confirm that we do not show on this map the potential impacts of nuclear waste on the environnement. This map only shows, as stated in the header, the "Live CO2 emissions of the European electricity consumption". Nothing more. Nothing less.

I was not requesting to factor in the impacts of nuclear waste, but the electricity (and therefore, CO2 emission) consumed to take care of the waste. The title is rather confusing, because the map does not show the “live CO2 emissions”. If I understand it correctly, it shows the carbon intensity (measured in different ways, as taken from IPCC report), or the “climate impact” scaled to the current electricity generation of the different electricity sources. “Live CO2 emissions” sounds like the actual, real time CO2 emission (i.e. what comes out of the coal fired plant’s stack).

> This being said, if you have any data to support the fact that plant decommission and waste management are particularly energy intensive, sufficiently to have an impact on the emission factors, I would be quite interested.

I have mentioned the uncertainties before. There are a lot of meta-analyses out there that partly include these stages, but if you are looking for a more integrated data set on different electricity sources, I would look into publications of widely recognized, independent NGOs. I am sure this work has been done before. But that being said, the uncertainties will remain, because complete decommissioning and long term storage has not been performed anywhere in the world yet.

I think it all boils down to the dilemma of an appropriate but accessible representation of the complicated science that produces a wide range of possible answers, depending on assumptions, methods, boundaries... and that is a challenging task of high responsibility. 

I personally do have a strong opinion about nuclear power as an electricity source, but that is not my motivation in this discussion. I looked at the map as a scientist and stumbled over the underlying data. 

No matter what decision you make on the data selection, most importantly, it should be clear at first sight what you are showing. If you could put a disclaimer on the map that clearly states where the data comes from, that would help the interpretation a lot.

Best wishes,
Elke

Manfred Lenzen 2008. Life cycle energy and greenhouse gas emissions of nuclear energy: A review. Energy Conversion and Management, 49 (8), 2178-2199

Benjamin K. Sovacool 2008. Valuing the greenhouse gas emissions from nuclear power: A critical survey. Energy Policy, 36 (8), 2950-2963

Ethan S. Warner and Garvin A. Heath 2012. Life Cycle Greenhouse Gas Emissions of Nuclear Electricity Generation: Systematic Review and Harmonization. Journal of Industrial Ecology, 16, S1, DOI: 10.1111/j.1530-9290.2012.00472.x

**Comment by martindaniel4** [comment]
Thanks for your thorough feedback Elke!

How would you emphasize more the data sources?

**Comment by ElkeBrandes** [comment]
I would put a disclaimer directly on the website, summarizing the README, to make it easier for the majority of visitors not familiar with github. It should clearly but briefly state the limitations, data sources and methods. 

I have fallen on my face for a related issue and link this as an example: [here is a website](http://mesonet.agron.iastate.edu/GIS/apps/profit/) of some estimated data we published recently. We added the disclaimer after we got feedback from farmers who zoomed in on their fields and said our data was wrong. Our bad - the website was useless without the information in the paper on how we came up with these estimates. With the disclaimer we're off the hook and the maps are shown in the right context.

**Comment by ThierryOllivero** [comment]
Elke,

Many thanks for this really detailed feedback on my message. I will reopen the issue because I would like our scientific expert @brunolajoie to give a look at the discussion.

Best regards,

Thierry

**Comment by brunolajoie** [comment]
Dear Elke, 

Thanks for your comments. I appreciate attention to details too. 

- About Nuclear LCA GHG intensity: We agree that Nuclear GHG intensity offers only a very limited degree of scientific consensus, due to the complexity of its value chain. However, at the moment in our code, we cannot refine GHG intensity beyond a single value in g/kWh, that could maybe be finetuned in a country by country basis. Therefore, what single reference value would you recommend to use other than 12, that could be justified scientifically? My personal guess is that, even if it was a bit higher, say between 20 and 50, it would not change significantly the color of a country, and therefore our message on ElectricityMap.

- Your disclamer seems a great idea. I agree github is a instant turn off for anyone not into coding. @corradio do you agree?

- About IPCC processes, you mentioned that they "referred to _selected_ peer reviewed scientific literature" --> I though they had no rights to reject any peer reviewed paper, provided it was in the scope of the analysis, and suggested by the authors to the IPCC board. Am I getting it wrong? Why would some study be "rejected"?

Best regards,
Bruno Lajoie

**Comment by corradio** [comment]
For the disclaimer, I refer to issue https://github.com/corradio/electricitymap/issues/264 which aims at showing more clearly that the emission intensities comes from IPCC 2014.

**Comment by ElkeBrandes** [comment]
Bruno,
putting a single value is tricky for nuclear energy because of the high uncertainties of the different stages, especially the downstream stages (after plant operation). I think it is common sense that these stages are disproportionately energy-intensive compared to other forms of electricity that do not produce highly hazardous waste. But it seems that because of this uncertainty, the downstream stages are often simply not included in the life cycle analysis. But when looking at a long term environmental problem like climate change, I think it is crucial to look at long term carbon emissions as well. 
Given the uncertainty, I would do a sensitivity analysis by plugging in different values within the range of the published meta-analyses: say, 50, 100, 200 g CO2 (you can pick some exemplary countries, one with high and one with low nuclear percentage). Then you know how sensitive your model is to this number, i.e., how relevant this input is.

Regarding the IPCC report, they have cited two references, but took the number of only one. Maybe they forgot to delete the Lenzen 2008 reference (with the higher values) during the review process. It looks like a selection of literature rather than explicit rejection of certain papers. At a different location in the report I read that they were using updated information. The Warner and Heath 2012 meta-analysis seems to be the latest published at that time, so maybe that is why they directly took their results. But as I wrote earlier, there are methodological flaws in the Warner and Heath paper, so the results should be interpreted with caution. 
I wish I could dive deeper into these LCA results, but unfortunately nobody pays me to do so... ;-)

Best wishes,
Elke

**Comment by ThierryOllivero** [comment]
Dear Elke,

I reopened the issue after seeing that you were positively trying to bring arguments in the discussion. Your last phrase makes me kind of regret it, I will come back to it.

First on this line:

> I think it is common sense that these stages are disproportionately energy-intensive compared to other forms of electricity that do not produce highly hazardous waste.

I agree, that compared to other process, these stages are energy-intensive. Mostly because there are basically no steps for the other sources. But if you compare the energy use for waste treatment to the total energy produced by what became the waste, this is absolutely negligible. Keep in mind  that the energy stored in 1 kg of enriched Uranium is equivalent to the energy stored in 3 million kilograms of coal.  (source: http://www.ems.psu.edu/~radovic/Chapter13.pdf) . So even if you spend a few GWh treating 1 kg of Uranium (which is probably much more than the reality), this is still a very very small fraction of what this kg of Uranium will have produced in its lifetime. So the CO2eq/kWh produced is close to 0.

Now coming to this part:

> I wish I could dive deeper into these LCA results, but unfortunately nobody pays me to do so... ;-)

Two possibilities:

1. I did not understand this part of your message, then the following lines are worthless
2. You suggest that we are being paid by someone to do this kind of work 

Since we are an open-source project and anyone in the World can read our messages, I cannot just assume that the truth is 1.

So let me get this straight: We are not being paid. At all. No one, no company, no organization, no lobby, not any other kind of group or persons pay us to work on the Electricity Map. This is something that we do mostly on our free time. And we quite enjoy to do so, quite frankly.

Some people use our Electricity Map to show that Renewable Energy is wonderful or terrible. Other that Coal is terrible. Other that Nuclear is wonderful or terrible. Good thing is, no one yet came with the idea that coal is wonderful. 

We, at Tomorrow, are only driven by our Mission Statement available on our website http://www.tmrow.co : 

> We want to propel humanity to a sustainable state of existence by quantifying, and making widely accessible, the impact of every choice we make.

This is it. Nothing more. Nothing less. And yes, you are allowed to find it naive, or ridiculous, or amazing, or revolutionary, or whatever.

Each of us is entitled to like or not like, believe in or not believe in, a technology or another. I am personally in favor of all technologies that have a limited impact on the environment and I acknowledge that most of them do have one (hydro has one when dams create huge lakes over villages, nuclear has one with the waste that we do not know how to treat , etc). I am also, still personally in favor of nuclear research, both for new technologies and new ways of treating the waste and I believe in extensive renewable development. My dream is that one day we can completely get rid of coal, gas and oil. If you have a good idea on how to get rid of nuclear after these 3, I would have absolutely no problem following it.

We are open to any discussion that will make our model more accurate and more precise. We acknowledged that the value given by IPCC might not be perfect, and we are open to discuss this topic. But if you are only here to troll about us being paid to display one thing or another, please find another playground. 

Best regards, 

Thierry Ollivero

**Comment by ElkeBrandes** [comment]
Dear Thierry,
I'm sorry that my sentence was misunderstood. I only referred to myself, noticing that there is lots of work out there that would be interesting to do, but that certain constraints (but mostly time) prevent me from doing it.
Best wishes,
Elke

**Comment by ThierryOllivero** [comment]
Dear Elke,

Thanks for clarifying this and sorry for my overreaction. Please consider that this was not directed to you personally.

We have sometimes been under "attack" from people insinuating on some Social Networks that we are being paid by lobby groups. Which can be quite frustrating given the time we spent on this the last 6 months :)

At least this gave me the opportunity to remind people of what we are, and what we are not. 

Have a nice day!

Thierry

**Comment by brunolajoie** [comment]
Dear all, thanks for this insightful discussion.

We learned that evaluating LCA emissions of nuclear power was highly uncertain and dependent on various parameters such as type of reactor, uranium mine used etc...therefore, highly country dependant.  However, IPCC's 12gCO2/kWh is not a significantly biased single value as we must pick one.

Therefore, I'm closing this specific issue, keeping in mind that we will be able to refine them further in the future, should we decide to look at country specific LCA or to split Operational emissions from Construction-related emissions of power plants. (but that's a different issue: #237)

Best, Bruno
